### ISSUE TITLE
Python error when loading On Deck

### ISSUE THREAD

**Issue by quietvoid** [croneter/PlexKodiConnect#1641]
## Help yourself
* I did try to restart Kodi :-)
* I checked the [PKC Frequently Asked Questions on the PKC wiki](https://github.com/croneter/PlexKodiConnect/wiki/faq)
* I did try to reset the Kodi database by going to `PKC Settings -> Advanced -> "Reset the database and optionally reset PlexKodiConnect"` and then hitting YES, NO
* I did check the [existing issues on Github](https://github.com/croneter/PlexKodiConnect/issues)

## Describe the bug
When loading (at launch or manually) the On Deck list, PKC shows an error notification.
The On Deck widget then disappears.

However, regular syncing works fine.

## To Reproduce
1. Launch Kodi
2. Wait for PKC to start syncing On Deck.
3. Error happens

## Expected behavior
There should not be an error, and the On Deck widget should be populated.

## You need to attach a KODI LOG FILE!
Debug log: [kodi.log](https://github.com/croneter/PlexKodiConnect/files/7232508/kodi.log)

Note: I am running Kodi 20 N built on 2021-09-23, not 19.x.

**Comment by croneter** [comment]
Traceback in your log file:
```
- NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
Error Type: <class 'ImportError'>
Error Contents: This platform lacks a functioning sem_open implementation, therefore, the required synchronization primitives needed will not function, see issue 3770.
Traceback (most recent call last):
  File "/data/user/0/org.xbmc.kodiDV/cache/apk/assets/python3.8/lib/python3.8/multiprocessing/synchronize.py", line 28, in <module>
    from _multiprocessing import SemLock, sem_unlink
ImportError: cannot import name 'sem_unlink' from '_multiprocessing' (unknown location)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/storage/emulated/0/Android/data/org.xbmc.kodiDV/files/.kodi/addons/plugin.video.plexkodiconnect/default.py", line 203, in <module>
    Main()
  File "/storage/emulated/0/Android/data/org.xbmc.kodiDV/files/.kodi/addons/plugin.video.plexkodiconnect/default.py", line 44, in __init__
    entrypoint.browse_plex(key=params.get('key'),
  File "/storage/emulated/0/Android/data/org.xbmc.kodiDV/files/.kodi/addons/plugin.video.plexkodiconnect/resources/lib/entrypoint.py", line 527, in browse_plex
    show_listing(xml, plex_type, section_id, synched, key)
  File "/storage/emulated/0/Android/data/org.xbmc.kodiDV/files/.kodi/addons/plugin.video.plexkodiconnect/resources/lib/entrypoint.py", line 250, in show_listing
    all_items = utils.process_method_on_list(widgets.generate_item, all_items)
  File "/storage/emulated/0/Android/data/org.xbmc.kodiDV/files/.kodi/addons/plugin.video.plexkodiconnect/resources/lib/utils.py", line 869, in process_method_on_list
    pool = ThreadPool()
  File "/data/user/0/org.xbmc.kodiDV/cache/apk/assets/python3.8/lib/python3.8/multiprocessing/pool.py", line 925, in __init__
    Pool.__init__(self, processes, initializer, initargs)
  File "/data/user/0/org.xbmc.kodiDV/cache/apk/assets/python3.8/lib/python3.8/multiprocessing/pool.py", line 196, in __init__
    self._change_notifier = self._ctx.SimpleQueue()
  File "/data/user/0/org.xbmc.kodiDV/cache/apk/assets/python3.8/lib/python3.8/multiprocessing/context.py", line 113, in SimpleQueue
    return SimpleQueue(ctx=self.get_context())
  File "/data/user/0/org.xbmc.kodiDV/cache/apk/assets/python3.8/lib/python3.8/multiprocessing/queues.py", line 336, in __init__
    self._rlock = ctx.Lock()
  File "/data/user/0/org.xbmc.kodiDV/cache/apk/assets/python3.8/lib/python3.8/multiprocessing/context.py", line 67, in Lock
    from .synchronize import Lock
  File "/data/user/0/org.xbmc.kodiDV/cache/apk/assets/python3.8/lib/python3.8/multiprocessing/synchronize.py", line 30, in <module>
    raise ImportError("This platform lacks a functioning sem_open" +
ImportError: This platform lacks a functioning sem_open implementation, therefore, the required synchronization primitives needed will not function, see issue 3770.
-->End of Python script error report<--
```

This is nothing PlexKodiConnect can solve. It's an issue how Python has been implemented for Kodi on your Kodi-distribution. Do try to use a stable Kodi version, even Kodi 18.9. 
```
Starting Kodi (19.2-ALPHA2 (19.90.101) Git:20210923-898ae42aa6-dirty). Platform: Android ARM 32-bit
```

**Comment by quietvoid** [comment]
Okay, I wasn't sure if PKC was using some new Python feature or something.
Thanks.

**Comment by quietvoid** [comment]
Since https://github.com/xbmc/xbmc/pull/20034 has been backported to v19 Matrix, this is going to be an ongoing problem on 19.2.

**Comment by croneter** [comment]
OK this is a PITA that Kodi causes this... 🙄 

I'll try to patch this on the PKC side

**Comment by croneter** [comment]
Should be fixed with PKC beta version 3.5.1

**Comment by quietvoid** [comment]
It is working for me now on 3.5.1.
Thanks!
